BD-1184: Clarify Braze account user permissions (#3015)
braze-inc/braze-docs: _docs/_user_guide/administrative/manage_your_braze_users/user_permissions.md

You can manage user permissions by group or on an individual basis using the User Permissions page.

![Manage User Permissions][89]

|Permission Name|Definition/Parameters|
|---|---|
|Admin|Has access to all available features, default setting for all new users. Can update company settings (company name and time zone), which Limited Users are unable to do.|

> |Access Campaigns, Canvases, Cards, Segments, Media Library| User can view campaign and Canvas performance metrics, create drafts of campaigns and Canvases, view News Feed, segments, templates and media, create templates, upload media, and view engagement reports. Note that the user cannot duplicate existing drafts or view existing live content. |

Reviewer: ```suggestion
|Access Campaigns, Canvases, Cards, Segments, Media Library| User can view campaign and Canvas performance metrics, create drafts of campaigns and Canvases, view drafts of News Feed, segments, templates and media, create templates, upload media, and view engagement reports. However, users with this permission cannot duplicate existing drafts or view existing live content. |
```
